Gemfile

```diff
@@ -25,6 +28,7 @@ gem "title"
 gem "uglifier"
 gem "unicorn"
 
+
```

Comment:
Is ` gem "rails_12factor"` missing?

I probably wasn't clear, sorry about that .. the `rails_12factor` includes the logging, so need to keep ` gem "rails_12factor"` but remove the other :)